docs/integrations/destinations/bigquery.md

```diff
@@ -107,6 +88,15 @@ You should now have all the requirements needed to configure BigQuery as a desti
 
 Once you've configured BigQuery as a destination, delete the Service Account Key from your computer.
 
+#### Uploading options
+There are 2 available options to upload data to bigquery `Standard` and `GCS Staging`.
+- `Standard` is option to upload data directly from your source to BigQuery storage. This way is faster and requires less resources than GCS one.
+  Please be aware you may see some fails for big datasets and slow sources, i.e. if reading from source takes more than 10-12 hours.
+  This is caused by the Google BigQuery SDK client limitations. For more details please check https://github.com/airbytehq/airbyte/issues/3549
+- `GCS Uploading (CSV format)`. This approach has been implemented in order to avoid the issue for big datasets mentioned above.
+  At the first step all data is uploaded to GCS bucket and then all moved to BigQuery at one shot stream by stream.
+  The destination-gcs connector is partially used under the hood here, so you may check its documentation for more details.
```

Comment:
```suggestion
  The `destination-gcs` connector is partially used under the hood here, so you may check its documentation for more details.
```

Reply:
maybe add link to destination-gcs to doc

Reply:
Added the link!